src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java

```diff
@@ -72,8 +72,6 @@ public void processByContainer(String containerName, List<Envelope> envelopes) {
             CloudBlobContainer blobContainer = blobManager.getContainer(containerName);
 
             envelopes.forEach(envelope -> processEnvelope(blobContainer, envelope));
-        } catch (URISyntaxException | StorageException exception) {
-            log.error("Unable to get client for {} container", containerName, exception);
```

Comment:
Removed, because:
* there's no need for special treatment of these two exceptions. Logging of `Exception` includes all the information
* log message not guaranteed to accurately describe the actual situation, as in the try clause there's more than just getting the client